examples/advanced-component-usage/sortable_table.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+
+"""
+A simple application demonstrating how to build a sortable dbc.Table using sorttable.js
+(https://www.kryogenix.org/code/browser/sorttable/), a small JS library to make an html table sortable.
+"""
+
+import dash
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+from dash.dependencies import Input, MATCH, Output, State
+
+DUMMY_STORE_TYPE = 'dummy-store'
+SORTABLE_TABLE_TYPE = 'sortable-table'
+COMPONENT_ID_INDEX = 1
+
+app = dash.Dash(__name__,
+                assets_folder='sorttable_assets',
+                external_stylesheets=[dbc.themes.BOOTSTRAP],
+                suppress_callback_exceptions=True)
+
+
+def create_table(num_rows):
+    accordion = html.Div(
+        [
+            dbc.Table(
+                [
+                    html.Thead(
+                        html.Tr([
+                            html.Th('Name'),
+                            html.Th('Number')
+                        ])
+                    ),
+                    html.Tbody([
+                        html.Tr([
+                            html.Td(5 * chr(ord('A') + i)),
+                            html.Td(num_rows - i)
+                        ]) for i in range(num_rows)
+                    ])
+                ],
+                id={'type': SORTABLE_TABLE_TYPE, 'id': COMPONENT_ID_INDEX},
+                bordered=True,
+                hover=True,
+                striped=True,
+            ),
+        ]
+    )
+
+    return accordion
+
+
+app.layout = html.Div([
+    dbc.Input(id='num_rows', placeholder='Number of table rows', type='text'),
+    html.Br(),
+    html.P(id='status-msg'),
+    html.Div(id='table-div'),
+    # The dcc.Store associated with the table must be created in the layout for the callback to work
+    dcc.Store(id={'type': DUMMY_STORE_TYPE, 'id': COMPONENT_ID_INDEX}, data=0),
```

Comment:
You are right, it is a late addition to the example and probably I should enhance it a little bit. The reason I did it is based on my experience with a real app where you generally have several tables where you want to activate sorting. I don't like the idea that you write one callback per table to activate the sorting, with the same callback code. A trivial way to achieve this is using pattern-matching callbacks. It took me some time to understand how to get it working because of this problem that a composite ID is not passed as a string to JS... so I found useful to mention the trick (3 JS lines) at the price of a small complexity increase. These lines are harmless with standard IDs and the callback code remains generic.